README.md

```diff
@@ -14,9 +14,24 @@ The test runner requires 2 parameters:
 go run . ./testdata/practice/passing outdir
 ```
 
+#### Run the package tests
+
+```bash
+go test .
+```
+
+#### Interactive Debug / REPL
+
+The original AST parsing code was developed using a Jupyter interactive Go REPL
+thanks to the [gophernotes project](https://github.com/gopherdata/gophernotes). Consult the gophernotes docs for installation instructions. Once installed, you should be able to view, run, and modify the provided debug code "live" without constantly recompiling:
```

Comment:
You could consider mentioning that this requires Python to be installed.

Reply:
Done! I also made the github actions change from #24. Also, in order to pass the new linter, I had to refactor the code into using a subpackage (`testrunner`). This is a more idiomatic way to structure the code, but I wanted to give you a heads up that this PR expanded since you initially reviewed it.

Reply:
Sure!